queries.md

```diff
@@ -2,22 +2,75 @@
 
 ## find all customers that live in London. Returns 6 records.
 
+```sql
+select * from customers
+where city like 'London'
+```
+
 ## find all customers with postal code 1010. Returns 3 customers.
 
+```sql
+select * from customers
+where postalcode like '1010'
+```
+
 ## find the phone number for the supplier with the id 11. Should be (010) 9984510.
 
+```sql
+select phone from suppliers
+where supplierid like '11'
+```
+
 ## list orders descending by the order date. The order with date 1997-02-12 should be at the top.
 
+```sql
+select * from orders
+order by orderdate desc
+```
+
 ## find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
 
+```sql
+select * from suppliers
+where length(suppliername) > 20
+```
+
 ## find all customers that include the word "market" in the name. Should return 4 records.
 
+```sql
+SELECT * FROM [Customers]
+where customername like '%market%'
+```
+
 ## add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
 
+```sql
+INSERT INTO customers (customername, contactname, address, city, postalcode, country)
+VALUES ("The Shire", "Bilbo Baggins", "1 Hobbit-Hole", "Bag End", "111", "Middle Earth");
+```
+
 ## update _Bilbo Baggins_ record so that the postal code changes to _"11122"_.
 
+```sql
+update customers
+set postalcode = '11122'
+where contactname like 'Bilbo Baggins'
+```
+
 ## list orders grouped by customer showing the number of orders per customer. _Rattlesnake Canyon Grocery_ should have 7 orders.
 
+```sql
+WITH working as (
+	select customerid, 
+    count(*) as "orders"
+		from orders
+			group by customerid)
+
+select Customers.CustomerName, working.orders
+    from working
+        inner join customers on working.customerid=customers.customerid
```

Comment:
This looks like it might be more complicated than it needs to be, but it gets the job done! I'd try to take advantage of aliases for tables if I were you (plus it just makes it harder to digest large queries if there are wordy table names in there):
```sql
WITH w as (
  select customerid, 
  count(*) as "orders"
  from orders
  group by customerid)
select c.CustomerName, w.orders
  from working
  inner join customers c on w.customerid=c.customerid
```